.github/workflows/test-install.yml

```diff
@@ -214,13 +226,11 @@ jobs:
         pytest -n auto --db-backend psql -m 'not nightly' tests/
 
     - name: Slack notification
-      # Always run this step (otherwise it would be skipped if any of the previous steps fail)
-      # but only if the `install` or `tests` steps failed.
+      # Run this step if any of the previous steps fail.
       # Don't run on PRs, the failure is clearly visible in GitHub UI.
       # Run only when the `secrets.SLACK_WEBHOOK` is available, which is not the case for forks.
       if: >-
-        always() &&
-        (steps.install.outcome == 'failure' || steps.tests.outcome == 'failure') &&
+        failure() &&
```

Comment:
This is a better way to express what we need: run this step iff any of the previous steps failed.

https://docs.github.com/en/actions/reference/workflows-and-actions/expressions#failure